packages/ses/src/tame-symbol-constructor.js

```diff
@@ -50,14 +50,20 @@ export const tameSymbolConstructor = () => {
     },
   });
 
+  const sharedSymbolDescs = getOwnPropertyDescriptors(SharedSymbol);
   const originalDescsEntries = entries(
     getOwnPropertyDescriptors(OriginalSymbol),
   );
   const descs = fromEntries(
```

Comment:
consider adding a comment like
`// Hermes has many non-configurable Symbol properties. We should support such platforms.`

Reply:
The JS spec says that well known symbols are non-configurable properties of the `Symbol` global. How is hermes different exactly?

Reply:
looking further at the source

```h
...
/// Represents a JS Symbol (es6).  Movable, not copyable.
/// TODO T40778724: this is a limited implementation sufficient for
/// the debugger not to crash when a Symbol is a property in an Object
/// or element in an array.  Complete support for creating will come
/// later.
class JSI_EXPORT Symbol : public Pointer {
...
```
https://github.com/facebook/hermes/blob/main/API/jsi/jsi/jsi.h#L515-L540

https://github.com/facebook/hermes/blob/main/include/hermes/VM/PrimitiveBox.h#L310-L347